docs/building.md

```diff
@@ -17,13 +17,59 @@ The latest working version tested is [0.24.0](https://gitlab.com/ligolang/ligo/-
 
 You can use the [`Makefile`](../Makefile) to build the LIGO contract by simply running:
 ```sh
-make out/baseDAO.tz
+make out/trivialDAO.tz
```

Comment:
This section is now somewhat odd, because it's about "Generation the contract code" but doesn't mention the fact that there are now multiple ways to do so...

IMO it should point this out and not only give the instructions for `trivialDAO`.

Reply:
Have changed